config/stage.json

```diff
@@ -18,6 +18,7 @@
 		"post-editor/iframe-preview": true,
 		"post-editor/live-image-updates": true,
 		"post-editor/pages": true,
+		"post-editor/contact-form": true,
```

Comment:
What's the reasoning for enabling this in staging? My impression of staging is that it's intended for features that are essentially shippable, and very close to/mirroring production readiness.